core/src/main/java/com/github/mizool/core/rest/errorhandling/behavior/InvalidBackendReplyExceptionBehavior.java

```diff
@@ -0,0 +1,49 @@
+package com.github.mizool.core.rest.errorhandling.behavior;
+
+import javax.servlet.http.HttpServletResponse;
+
+import org.kohsuke.MetaInfServices;
+
+import com.github.mizool.core.exception.InvalidBackendReplyException;
+import com.github.mizool.core.rest.errorhandling.ErrorHandlingBehavior;
+import com.github.mizool.core.rest.errorhandling.LogLevel;
+
+@MetaInfServices
+public class InvalidBackendReplyExceptionBehavior implements ErrorHandlingBehavior
+{
+    @Override
+    public Class<? extends Throwable> getThrowableClass()
+    {
+        return InvalidBackendReplyException.class;
+    }
+
+    @Override
+    public boolean includeErrorId()
+    {
+        return true;
+    }
+
+    @Override
+    public boolean includeDetails()
+    {
+        return true;
```

Comment:
This has changed with regards to the old `DefaultWhiteList`. Was that intentional or copy and paste?

Reply:
This was intentional, as the old behaviour was one of the motivations behind this change. Just to control the status code, we whitelisted this exception, resulting in us being blind in cases of backend to backend communication breaking down.